src/main/java/net/consensys/orion/cmd/Orion.java

```diff
@@ -518,6 +500,51 @@ public void run(final PrintStream out, final PrintStream err, final Config confi
     isRunning.set(true);
   }
 
+  private void applyServerCertChain(final HttpServerOptions options, List<Path> certChain) {
+    if (!certChain.isEmpty()) {
```

Comment:
Not sure I agree - i.e. its nice to keep this whole check together - i.e. even if we move this check outside, we still need to do it inside I think...in case someone calls this function elsewhere with an empty list...

Reply:
I believe Jason's point is that someone might have the impression that we are applying a certificate chain when we aren't because the list is empty.

I wonder if renaming the method to `applyServerCertChainIfAvailble` or something similar would solve this.

Either way, I think keeping the check within this method yields cleaner code on the top level method.